overlord/servicestate/quota_handlers.go

```diff
@@ -69,6 +69,10 @@ type QuotaControlAction struct {
 	ParentName string `json:"parent-name,omitempty"`
 }
 
+func init() {
+	snapstate.AddSnapToQuotaGroup = AddSnapToQuotaGroup
```

Comment:
this is another preexisting pattern, I don't want to diverge punctually on it here. We might revisit the pattern at a later point

Reply:
to be clearer, I'm not against using Register instead of Add in the functions in the conflict mechanism, just that it should be its own change. It is also worth remembering that golang has types, so it's partly the case the purpose of a function can be understood from name + argument types and not only name.

Reply:
(sorry I added a comment (now deleted) to the wrong comment thread)